packages/bbui/src/Modal/ModalContent.svelte

```diff
@@ -60,6 +61,7 @@
         {/if}
         {#if showConfirmButton}
           <Button
+            {warning}
```

Comment:
`$$restProps` here already passes in any other props to the button deliberately, to avoid needing to do things like add all the button props to the `ModalContent` component too.  So I think this should work fine already without adding the extra prop 👍

Reply:
Oh, completely missed that! 👍